frontend/src/renderer/components/Charts/AtomicHeatMap.svelte

```diff
@@ -48,84 +51,113 @@
         "#ff1200"
     ];
 
+    const grayScale = [
+        "#ffffff",
+        "#efefef",
+        "#d3d3d3",
+        "#c1c1c1",
+        "#b5b5b5",
+        "#9e9e9e",
+        "#898989",
+        "#767676",
+        "#585858",
+        "#4c4c4c",
+        "#4a4a4a",
+        "#4c4c4c",
+        "#383838",
+        "#383838",
+        "#313131",
+        "#2a2a2a",
+        "#1c1c1c",
+        "#191919",
+        "#171717",
+        "#000000"
+    ];
+
     function ranges() {
         const result = [];
         for (let i = 0; i < 20; i++) {
             result.push({
                 from: i/2,
                 to: i/2 + .5,
-                color: colors[i]
+                color: visionDeficiencyMode ? grayScale[i] : colors[i]
             });
         }
         result[result.length - 1].to = Number.MAX_SAFE_INTEGER;
         return result;
     }
 
-    const options = {
-        series: getData(),
-        chart: {
-            height: '600px',
-            width: '600px',
-            type: 'heatmap',
-            toolbar: {
-                // Hamburger menu which has exports such as CSV etc.
-                // I have had issues displaying this, I believe some unrelated global CSS is causing issues
+    const getOptions = function() {
```

Comment:
just changed this to a function instead of a constant so it can update automatically when mode is switched